packages/shared/lib/participation/staking.ts

```diff
@@ -296,3 +294,15 @@ export const isStakingForAssembly = (participations: Participation[]): boolean =
 
     return eventIds.includes(ASSEMBLY_EVENT_ID)
 }
+
+/**
+ * Determines whether staking is currently in the pre-stake or holding period
+ *
+ * @method isStakingPossible
+ *
+ * @param {ParticipationEventState} stakingEventState
+ *
+ * @returns {boolean}
+ */
+export const isStakingPossible = (stakingEventState: ParticipationEventState): boolean =>
+    stakingEventState !== ParticipationEventState.Ended && stakingEventState !== ParticipationEventState.Inactive
```

Comment:
We should probably also add a third check for `ParticipationEventState.Upcoming`, or we could check for `Commencing` and `Holding`.